crates/ra_ide_db/src/feature_flags.rs

```diff
@@ -54,6 +54,7 @@ impl Default for FeatureFlags {
         FeatureFlags::new(&[
             ("lsp.diagnostics", true),
             ("completion.insertion.add-call-parenthesis", true),
+            ("completion.insertion.add-argument-sippets", true),
```

Comment:
Sippets

Reply:
Good catch, fixed in https://github.com/rust-analyzer/rust-analyzer/pull/3500